docs-v2/reference/scripts.mdx

```diff
@@ -202,6 +202,28 @@ await nango.triggerAction('<ACTION-NAME>', { 'custom_key1': 'custom_value1' });
 
 _TbD_
 
+# Get Integration
+
+Returns the current integration information
+
+```js
+await nango.getIntegration();
+```
+
+With credentials
+
+```js
+await nango.getIntegration({ include: ['credentials'] });
+```
+
+**Parameters**
+
+See `GET /integrations/{uniqueKey}` query parameters: [documentation](/reference/api/integration/get)
```

Comment:
Do we use this `see ENDPOINT` pattern for any other method? Should we?

Reply:
I don't think so, and I don't know.
Honestly, those docs are annoying to maintain and got lazy but it's also the first one where the api and the method matches perfectly